docs/arch/adr-11.md

```diff
@@ -0,0 +1,48 @@
+
+Now that we've adopted Grommet as general React component library, Grommet also provides a React [markdown](https://v2.grommet.io/markdown) renderer ([code](https://github.com/grommet/grommet/blob/master/src/js/components/Markdown/Markdown.js)). Grommet's markdown component uses [markdown-to-jsx](https://github.com/probablyup/markdown-to-jsx) which instead converts markdown to React components to use instead of relying on `dangerouslySetInnerHTML`. However, after extensive evaluation, `markdown-to-jsx` does not have the plugin eco-system that we need and so we would need to rewrite a lot of customizations to get to basic parity with what we already support. This defeats the purpose of reducing the maintenance of our own code for common markdown support.
+
+## Decision
+
+We will make a new `Markdownz` React component that will be a part of the Zooniverse React component library. This new component will be built using [`remark`](https://github.com/remarkjs/remark). Remark is a popular markdown rendering library with a good plugin eco-system. It is supported by Zeit, which also supports Next.js, the server-side rendering library we have decided upon. 
+
+Here is how markdown-it's plugins will map to remark's plugins:
+
+|markdown-it plugin/custom plugin|remark plugin/custom plugin|notes|
+|--------------------------------|---------------------------|-----|
+|markdown-it-emoji|remark-emoji|remark-emoji does not support emoticons like `:-)` but does gemojis like `:smile:`|
+|markdown-it-sub|remark-sub-super||
+|markdown-it-sup|remark-sub-super||
+|markdown-it-footnote|built in|Remark supports this and can be enabled by passing `footnote: true` into its settings object|
+|markdown-it-imsize|N/A|This has been replaced by leveraging the component customization that remark-react supports. For `img`, we have defined a custom function that will set the `width` and `height` props on the Image component if the sizing syntax is defined in the alt tag of the markup. This is in contrast to the sizing syntax originally being defined in the src markup. We do not want to modify the sanitization remark-react does on source urls, so instead we have moved support of syntax to the alt tag area of the markup|
+|markdown-it-video|deprecating|We are deprecating this because we don't want project owners embedding youtube videos with ads|
+|markdown-it-table-of-contents|remark-toc|This works instead by looking for a heading that has case insensitive `table of contents`, `toc`, or `table-of-contents`|
+|markdown-it-anchor|N/A|Remark has basic anchor support.|
+|twemoji|N/A|Do we really need to use images of Twitter's emojis? Unicode support for emojis is fairly ubitiquous now.|
+|markdown-it-html5-embed|N/A|This has been replaced by leveraging the component customization that remark-react supports. For `img`, we define a custom function that returns a video instead of an image of the src is a video mime-type|
```

Comment:
We currently support audio embedding too, and I think audio projects have used that, so audio should be mentioned along with video.